vendor/bat-native-ads/README.md

```diff
@@ -62,7 +62,7 @@ void ChangeLocale(
 
 `ClassifyPage` should be called when a page has loaded in the current browser tab, and the HTML is available for analysis
```

Comment:
Function name in comment needs an update...

Reply:
Fixed